internal/future/statestore/redis.go

```diff
@@ -73,8 +73,9 @@ func newRedis(cfg config.View) (Service, error) {
 	// always returns a valid connection, and will just fail on the first
 	// query: https://godoc.org/github.com/gomodule/redigo/redis#Pool.Get
 	redisConn := pool.Get()
-	defer redisCloser(redisConn.Close)
-	_, err := redisConn.Do("SELECT", "0")
+	defer handleConnectionClose(redisConn, &err)
```

Comment:
I think it's fine to return without an error if the connection close fails. The operation succeeded.